libs/core/src/scss/interaction-state/_hover.scss

```
@@ -0,0 +1,54 @@
+@use 'sass:color';
+@use 'sass:meta';
+@use '../utils';
+
+// TODO: Accessibility - Loudness must respect user preferences
+// See https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-contrast
+// TODO: Await UX decision on default loudness and update
+$default-loudness: utils.get-loudness('loud');
+
+@mixin apply-hover($background-color, $color, $property, $loudness: null) {
```

Is it possible to do something equivalent to this: 
```suggestion

@mixin apply-hover($background-color, $color, $property, $loudness: utils.get-loudness('loud')) {
```

or perhaps: 
```suggestion
$default-loudness: utils.get-loudness('loud');

@mixin apply-hover($background-color, $color, $property, $loudness: $default-loudness) {
```

---

Initially `$default-loudness` was a private member `$_default-loudness` but it had to be public so it could be exposed to e.g. `button.component.scss`. However, it's not being used outside of this module at the moment, so maybe it can be changed again.

I'll see what I can come up with.